lib/specinfra/command/darwin/base/package.rb

```diff
@@ -33,6 +33,10 @@ def install(package, version=nil, option='')
       cmd = "brew install #{option} '#{package}'"
     end
 
+    def remove(package, version=nil, option='')
```

Comment:
`version` is unnecessary. Define the method as same as other platforms.

See https://github.com/mizzy/specinfra/blob/f7d4c91bcc05ca1327c09e77424dd071f62a9474/lib/specinfra/command/redhat/base/package.rb#L26